test/common/verifier.go

```diff
@@ -49,6 +54,27 @@ func NewVerifier(log logr.Logger,
 	}
 }
 
+// WithConsistentlyArgs specifies the arguments to pass to
+// `Consistently` calls made by the Verifier.
+func (v *Verifier) WithConsistentlyArgs(args ...interface{}) *Verifier {
+	v.consistentlyArgs = args
+	return v
+}
+
+// WithEventuallyArgs specifies the arguments to pass to
+// `Eventually` calls made by the Verifier.
+func (v *Verifier) WithEventuallyArgs(args ...interface{}) *Verifier {
+	v.eventuallyArgs = args
+	return v
+}
```

Comment:
IIUC, the motivation for this change is to have different timeout and poll interval for e2e and TM tests. 
However, we found these options confusing as it is not very clear to the reader for which assertions these timeouts/poll intervals are used. Tomorrow, you may need to add another assertion in the test and how we will make the diff between the eventuallyArgs used for assertion foo and the eventuallyArgs used for assertion bar.

We would rather suggest to hard-code the higher timeouts/poll intervals. The e2e tests will be simply faster and won't need to whole time. Or, is there are any strong reason to have different values for the different types of tests?